.travis.yml

```diff
@@ -19,11 +19,12 @@ cache:
   directories:
     - $HOME/.stack
     - .stack-work
+    - back/.stack-work
```

Comment:
Does  `stack-work` stored in both directories?

Reply:
Yep. (Unfortunately.)